climt/_components/surface_ice.py

```diff
@@ -296,18 +305,24 @@ def calculate_new_ice_temperature(self, rho, specific_heat, kappa,
         a_sub = np.zeros(num_layers)
         a_sup = np.zeros(num_layers)
 
-        K_bar = 0.25*(kappa[2:] + kappa[:-2]) + 0.5 * kappa[1:-1]
-        K_mid = 0.5*(kappa[1:] + kappa[:-1])
+        # K_bar = 0.25*(kappa[2:] + kappa[:-2]) + 0.5 * kappa[1:-1]
```

Comment:
Remove commented lines - these will be in the git history if you need them later!